incubator/fluentd/values.yaml

```diff
@@ -9,11 +9,15 @@ image:
   #   - secret1
   #   - secret2
 
-output:
-  host: elasticsearch-client.default.svc.cluster.local
-  port: 9200
-  buffer_chunk_limit: 2M
-  buffer_queue_limit: 8
+env:
+  open:
```

Comment:
`configMap` would be more clear

Reply:
@osterman env shows that we config container with environment variables

Reply:
I agree with @osterman.
Config maps are clearer.

Reply:
@osterman @rendhalver can you provide the example of the structure that would you prefer?
I can not get what is the problem